records/test2.yaml

```diff
@@ -0,0 +1,24 @@
+Collection Method: 
+  - Laboratory
+Country of Origin:
+  - International
+Developer: Testing
+Discipline:
+  - Materials
+Funding: WPTO
```

Comment:
The `Funding` field is a selection from list. Currently, the only valid value is `WPTO`, so this should be:

```
Funding:
  - WPTO
```